lua/configs/lsp/handlers.lua

```diff
@@ -34,6 +34,10 @@ astronvim.lsp.on_attach = function(client, bufnr)
         ["]d"] = { function() vim.diagnostic.goto_next() end, desc = "Next diagnostic" },
         ["gl"] = { function() vim.diagnostic.open_float() end, desc = "Hover diagnostics" },
       },
+      v = {
+        ["<leader>la"] = { function() vim.lsp.buf.range_code_action() end, desc = "Range LSP code action" },
+        ["<leader>lf"] = { function() vim.lsp.buf.range_formatting() end, desc = "Range Format code" },
```

Comment:
Could you uncapitalize the word "format" here

Reply:
Sure thing! Just fixed - hope it's all good now